website/src/main/tut/docs/rules/DisableSyntax.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: docs
+title: Disable
+---
+
+# DisableSyntax
+
+_Since 0.5.4_
+
+This rule reports errors when a "disallowed" syntax is used.
+
+Example:
+
+```scala
+MyCode.scala:7: error: [DisableSyntax.xml] xml is disabled.
+  <a>xml</a>
+  ^
+```
+
+## Configuration
+
+By default, this rule does allows all syntax.
+
+It contains the following elements:
+
+* keywords such as: `null, throw, var, return`
+* carriage return aka <CR> (windows uses <CR><LF>)
+* semicolons (`;`)
+* tabs
+* xml literals
+
+To disallow a syntax:
+
+```
+DisableSyntax {
+  keywords = [
+    abstract
+    case
+    catch
+    class
+    def
+    do
+    else
+    enum
+    extends
+    false
+    final
+    finally
+    for
+    forSome
+    if
+    implicit
+    import
+    lazy
+    match
+    macro
+    new
+    null
+    object
+    override
+    package
+    private
+    protected
+    return
+    sealed
+    super
+    this
+    throw
+    trait
+    true
+    try
+    type
+    val
+    var
+    while
+    with
+    yield
+  ]
+  carriageReturn = true
+  semicolons = true
+  tabs = true
+  xml = true
```

Comment:
How hard would it be to support arbitrary regexp? 

Also, make it possible to add a custom message/explanation for each kind? 

We can leave it for another pr or release, just wondering

Reply:
It would be nice indeed: 
https://github.com/apache/spark/blob/7f2e62ee6b9d1f32772a18d626fb9fd907aa7733/scalastyle-config.xml#L206-L215

Reply:
https://github.com/scalacenter/scalafix/issues/436